src/app/pages/home/home.component.html

```diff
@@ -1,11 +1,18 @@
-<div class="flex items-center justify-center max-w-[1000px] max-h-[680px] w-full h-full">
+<div class="centered-content">
     <div class="flex w-full h-full max-h-96 justify-center">
         <div class="bg-white w-64 flex flex-col py-2 rounded-l-md relative">
             <h1 class="text-black p-2 font-extrabold">Projects</h1>
             <div class="list-menu container flex items-center flex-col flex-grow-1">
-                <a class="item flex items-center justify-center px-2 py-4 rounded-md w-full text-[#5f5f5f] mb-1
+                <!-- <a class="item flex items-center justify-center px-2 py-4 rounded-md w-full text-[#5f5f5f] mb-1
                      hover:bg-pink-400 active:bg-pink-600 active:text-red-200">
                     <p>Project 1</p>
+                </a> -->
```

Comment:
Should never leave commented code in pull requests if there is not a good reason for it